compiler/rustc_lint/src/deref_into_dyn_supertrait.rs

```diff
@@ -52,9 +52,9 @@ declare_lint! {
     };
 }
 
-declare_lint_pass!(DerefIntoDynSupertrait => [DEREF_INTO_DYN_SUPERTRAIT]);
+declare_lint_pass!(DerefIntoDynSyncSendSupertrait => [DEREF_INTO_DYN_SUPERTRAIT]);
```

Comment:
A bit aggressive with the search and replace here.

Reply:
yea that's my mistake